commit 260f3df91cf3f4c271d3a66f0a3c8db1086e63c2
Author: Alexander Larsson <alexl@redhat.com>
Date:   Mon Jan 23 16:28:48 2017 +0100

    system-helper: Add User=root to service file
    
    This is needed for systemd-less activation on the system bus.

diff --git a/system-helper/org.freedesktop.Flatpak.SystemHelper.service.in b/system-helper/org.freedesktop.Flatpak.SystemHelper.service.in
index ed166ee..a6c7715 100644
--- a/system-helper/org.freedesktop.Flatpak.SystemHelper.service.in
+++ b/system-helper/org.freedesktop.Flatpak.SystemHelper.service.in
@@ -2,3 +2,4 @@
 Name=org.freedesktop.Flatpak.SystemHelper
 Exec=@libexecdir@/flatpak-system-helper@extraargs@
 SystemdService=flatpak-system-helper.service
+User=root
